utils/tracking_id.lua

```diff
@@ -1,37 +1,40 @@
--- Put functions in this file to use them in several other scripts.
--- To get access to the functions, you need to put:
--- require "my_directory.my_file"
--- in any script using the functions.
-
+local screens = require "utils.screens"
 
 local M = {
 
 	--Navigation Events
-	[hash("debug")] = 201,
-	[hash("are_you_sure")] = 202,
-	[hash("info")] = 203,
-	[hash("input")] = 204,
-	[hash("move_info")] = 205,
-	[hash("moves_scrollist")] = 206,
-	[hash("pick_name")] = 207,
-	[hash("random_pokemon")] = 208,
-	[hash("scrollist")] = 209,
-	[hash("sorting")] = 210,
-	[hash("transfer_pokemon")] = 211,
-	[hash("about")] = 212,
-	[hash("add")] = 213,
-	[hash("edit")] = 214,
-	[hash("party")] = 215,
-	[hash("profiles")] = 216,
-	[hash("splash")] = 217,
-	[hash("storage")] = 218,
-	[hash("generate_pokemon")] = 219,
-	[hash("status_effects")] = 220,
-	[hash("import_pokemon")] = 221,
-	[hash("pokedex")] = 222,
-	[hash("settings")] = 223,
-	[hash("fakemon")] = 224,
-	[hash("moves_confirm")] = 225,
+	[screens.DEBUG] = 201,
```

Comment:
I also changed this in my networking script. The merge shouldn't be bad, but maybe you'd want to do the same thing instead.

I gave tracking_id an init function and changed this [blah] = 20x into a flat array, then the init function loops through the array and assigns each a number. That way you don't need to increment a number every time, you just add the screen to the array.

Reply:
Yeah, probably a good idea. _Could_ make it a bit harder to verify which screen is which id though

Reply:
Leaving it as is. Have decided to remove all the tracking except the error reporting anyway.